src/Search/Api/Query/RamlRequirement.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\Search\Api\Query;
+
+trait RamlRequirement
+{
+    public static $root = __DIR__.'/../../../../vendor/elife/api/dist/samples/';
```

Comment:
At the moment the unit tests use them, and happy to replace with puli, but not exactly sure how to go about that. (I remember asking about the puli paths for the raml)

As for this part of the site, yes its only for the interim while we are getting the ES instance set up.